ECommerce.API/Controllers/ProductsController.cs

```diff
@@ -0,0 +1,106 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using ECommerce.DataAccess;
+using ECommerce.Entities;
+
+namespace ECommerce.API.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class ProductsController : ControllerBase
+    {
+        private readonly EShopContext _context;
+
+        public ProductsController(EShopContext context)
+        {
+            _context = context;
+        }
+
+        // GET: api/Products
+        [HttpGet]
+        public async Task<ActionResult<IEnumerable<Product>>> GetProducts()
+        {
+            return await _context.Products.ToListAsync();
```

Comment:
Should apply exception handling in API layer to return descriptive message to front end ..
Also we need to return OK status code in case of success

Reply:
Done